iina/MPVController.swift

```diff
@@ -1080,6 +1080,9 @@ fileprivate func mpvGetOpenGLFunc(_ ctx: UnsafeMutableRawPointer?, _ name: Unsaf
 
 fileprivate func mpvUpdateCallback(_ ctx: UnsafeMutableRawPointer?) {
   let layer = unsafeBitCast(ctx, to: ViewLayer.self)
```

Comment:
This isn't part of your changes, but I feel like `unsafeBitCast(_:to:)` is incorrect here. Perhaps this should use `UnsafeMutableRawPointer.assumingMemoryBound(to:)`?